src/telemetry/mod.rs

```diff
@@ -0,0 +1,48 @@
+pub mod metrics {
+    use metrics::describe_counter;
+    use metrics::Unit;
+
+    pub const REQUESTS_RECEIVED: &str = "requests.received";
+
+    /// Registers metrics used in the system with the metrics recorder.
+    ///
+    /// ## Panics
+    /// Panic if there is no recorder installed
+    pub fn register() {
+        assert!(
```

Comment:
is this equivalent to `assert!(metrics::try_recorder().is_some())`?

Reply:
yep I think it is the same